content-node/docker-compose/setup.sh

```diff
@@ -237,48 +201,24 @@ check_ports_and_ip() {
             log_success "Port $port is available"
         fi
     done
-
-    # Check if ports are accessible from outside
```

Comment:
i forgot why we added this. what's this for?

Reply:
This was a test we had to see if ports are actually open, don't think this is necessary. For example when someone is setting up a node, to verify that port 80 or port 443 is open.